### DISCUSSION TITLE
Supporting scripts in Magick.NET (e.g. this embroidery one)

### DISCUSSION THREAD

**Discussion by avisra** [dlemstra/Magick.NET#1348] [Help]
http://www.fmwconcepts.com/imagemagick/embroidery/index.php

Take, for example, the following script from the link above which generates a embroidery-look given a graphic. What would it take to make a script like this usable from .NET?

**Comment by dlemstra** [comment]
You can find that script translated to .NET here: https://github.com/dlemstra/ImageMagick.FredsScripts.NET. But you cannot use this in a commercial project without paying for a license.
